phalcon/db/dialectinterface.zep

```diff
@@ -111,7 +111,7 @@ interface DialectInterface
 	/**
 	 * Generates SQL to create a table
 	 */
-	public function createTable(string! tableName, string! schemaName, array! definition) -> string;
+	public function createTable(string! tableName, string! schemaName, array! definition);
```

Comment:
This method used as a "factory method". So it should return a string

Reply:
this one I did because of https://github.com/phalcon/cphalcon/blob/master/phalcon/db/dialect/postgresql.zep#L339 .  
The method there returns string there 100% time ( as far as I can see ) however I have never working with postgres so I am scared to touch anything there. But it seems like the postgres class can be changed to return string only ?